src/PockyBot.NET/Services/Triggers/Ping.cs

```diff
@@ -13,10 +14,10 @@ internal class Ping : ITrigger
 
         public string[] Permissions => new string[0];
 
-        public Message Respond(Message message) {
-            return new Message {
+        public Task<Message> Respond(Message message) {
```

Comment:
Maybe https://docs.microsoft.com/en-us/dotnet/api/system.threading.tasks.valuetask-1?view=netcore-3.0 ?

Reply:
why?

Reply:
> Only if performance analysis proves it worthwhile should a `ValueTask<TResult>` be used instead of a `Task<TResult>`.

I would suspect that as no performance analysis has been performed, and as PockyBot is not an especially performance-limited service, it is probably best to continue using a `Task<Message>`.

Reply:
Just something that might be worth trying when mixing sync calls and async calls in the same interface. I think I've read somewhere that almost always ValueTask should be preferred over Task but since its a new thing I haven't really tried it yet